configure.ac

```diff
@@ -360,6 +372,12 @@ AM_COND_IF([ASM], [
     AM_COND_IF([X86], [
         AC_DEFINE(ARCH_X86, 1, [targeting a 32- or 64-bit x86 host architecture])
     ])
+    AM_COND_IF([X86_64], [
+        AC_DEFINE(ARCH_X86_64, 1, [targeting a 64-bit x86 host architecture])
+    ])
```

Comment:
This C macro appears to be wholly unused, so adding it only bloats `configure.ac`.

Reply:
It would be needed by `checkasm.c`. Moved to relevant checkasm commit.